### ISSUE TITLE
newBoxShape(sx,sy,sz) creates uniform sized box?

### ISSUE THREAD

**Issue by eigenbom** [bjornbytes/lovr#630]
I noticed `lovr.physics.newBoxShape` was creating a uniformly sized box when giving it a non-uniform scale. E.g.,

```lua
local box = lovr.physics.newBoxShape(widget.scale[1], widget.scale[2], widget.scale[3])

print("newBoxShape: ", widget.scale[1], widget.scale[2], widget.scale[3])
print("dimensions: ", box:getDimensions())

box:setDimensions(widget.scale[1], widget.scale[2], widget.scale[3])
print("after set dimensions: ", box:getDimensions())
```

will print this (note dimensions is incorrect)

```
newBoxShape:    0.31398598806816        1       1
dimensions:     0.31398597359657        0.31398597359657        0.31398597359657
after set dimensions:   0.31398597359657        1       1
```

Workaround is to use setDimensions()

ref: https://lovr.org/docs/lovr.physics.newBoxShape

**Comment by bjornbytes** [comment]
This was already fixed in 3b3d2d4e7723dc95ca4482e4830ecc314e155fb7

**Comment by eigenbom** [comment]
👍
